examples/hello_world/module/cpp/hello_world.cc

```diff
@@ -70,8 +70,16 @@ WASM_EXPORT void oak_main(uint64_t grpc_in_handle) {
       return;
     }
 
-    uint64_t rsp_handle;
-    channel_read(grpc_in_handle, buf, sizeof(buf), &actual_size, &rsp_handle, 1, &handle_count);
+    uint64_t handles[2];
+    channel_read(grpc_in_handle, buf, sizeof(buf), &actual_size, handles, 2, &handle_count);
```

Comment:
Here the buffer should be zero-length, right? It is only needed below for the next read, and in that case the handle space can be zero-length. Or am I misunderstanding things?
(I know it would work either way, but it being an example, it makes sense for constants to be explicit, or otherwise documented).

Reply:
Done.